docs/source/multidc/multidc.md

```diff
@@ -0,0 +1,456 @@
+
+### Networking
+
+Since this guide assumes interconnectivity over PodIPs of the Kubernetes clusters, we are going to configure the ScyllaDB cluster's nodes to communicate over PodIPs.
+This is enabled by a subset of `exposeOptions` specified in ScyllaCluster API, introduced in v1.11.
+
+For this particular setup, we define the ScyllaClusers as follows:
+```yaml
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+spec:
+  exposeOptions:
+    nodeService:
+      type: Headless
+    broadcastOptions:
+      clients:
+        type: PodIP
+      nodes:
+        type: PodIP
+```
+
+However, other configuration options allow for the manual deployment of multi-datacenter ScyllaDB clusters in different network setups. For details, refer to [Exposing ScyllaClusters](../exposing.md) in ScyllaDB Operator documentation.
+
+#### Deploy a multi-datacenter ScyllaDB Cluster
+
+#### Using context
+
+We are going to specify contexts for `kubectl` commands used throughout the guide.
+To retrieve the context of your current cluster, run:
+```shell
+kubectl config current-context
+```
+
+Save the contexts of the two clusters, which we are going to deploy the datacenters in, as `CONTEXT_DC1` and `CONTEXT_DC2` environment variables correspondingly.
+
+#### Deploy the first datacenter
+
+First, run the below command to create a dedicated 'scylla' namespace:
+```shell
+kubectl --context "${CONTEXT_DC1}" create ns scylla
+```
+
+For this guide, we are assuming that your cluster is running in `us-east-1` region and the nodes dedicated to running ScyllaDB nodes are running in zones `us-east-1a`, `us-east-1b` and `us-east-1c` correspondingly. If that is not the case, adjust the manifest accordingly.
+
+``` caution::
+    The ``.spec.name`` fields of the ScyllaCluster objects have to be consistent across all datacenters being deployed.
+    The names of the datacenters, specified in ``.spec.datacenter.name``, have to be unique across the entire multi-datacenter cluster. \
+    
+    
+    For more information see `Create a ScyllaDB Cluster - Multi Data Centers (DC) <https://opensource.docs.scylladb.com/stable/operating-scylla/procedures/cluster-management/create-cluster-multidc.html>`_ in ScyllaDB documentation.
+```
+
+Save the ScyllaCluster manifest in `dc1.yaml`:
+```yaml
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+metadata:
+  name: scylla-cluster
+  namespace: scylla
+spec:
+  version: 5.2.7
+  agentVersion: 3.1.2
+  cpuset: true
+  exposeOptions:
+    broadcastOptions:
+      clients:
+        type: PodIP
+      nodes:
+        type: PodIP
+    nodeService:
+      type: Headless
+  datacenter:
+    name: us-east-1
+    racks:
+    - name: a
+      members: 1
+      storage:
+        storageClassName: scylladb-local-xfs
+        capacity: 1800G
+      resources:
+        limits:
+          cpu: 7
+          memory: 56G
+      placement:
+        nodeAffinity:
```

Comment:
for prod, we also need anti-affinity so they don't land on the same node with 2+ members

Reply:
I guess it's not fatal with racks but still better to spread them, at least as `preferredDuringSchedulingIgnoredDuringExecution` if not required

Reply:
(using `topologyKey: kubernetes.io/hostname`)

Reply:
should this be covered in this doc (this manifest specifically) or in the general guide?

Reply:
in every manifest and you have one here. I suppose it's more a question whether we keep piling up the deficiency or start fixing them at least in new manifests.

Reply:
Yup, but that's a more extensive topic, which I suppose is out of scope for this doc. I'll fix it in the new manifests

Reply:
done